tests/main/lxd-try/task.yaml

```diff
@@ -28,8 +28,12 @@ prepare: |
       lxd.lxc exec ubuntu -- sh -c "echo https_proxy=$https_proxy >> /etc/environment"
   fi
 
+  # wait for the container to be fully up
+  gretry -n 30 sh -c 'lxd.lxc exec ubuntu -- systemctl is-system-running | MATCH "(running|degraded)"'
```

Comment:
```suggestion
  retry -n 30 sh -c 'lxd.lxc exec ubuntu -- systemctl is-system-running | MATCH "(running|degraded)"'
```

Reply:
omg, must have been a mistyped C-g right before I committed everything. Thanks for fixing!